google-cloud-tools-plugin/change_notes.html

```diff
@@ -7,6 +7,28 @@
 </p>
```

Comment:
Should we also fix #2262 here and remove the paragraph about minimum Cloud SDK version since the default option is always the latest Managed SDK?

Reply:
yes. I'll put out a separate PR for that.